scripts/choco-install.ps1

```diff
@@ -0,0 +1,25 @@
+# Add the cached packages with source priority 1 (Chocolatey community is 0)
```

Comment:
Test for the existence of required env variables before running the script for sanity checking.

Like this:

```
if ($env:ChocolateyInstall -eq $null) {
  [Console]::Error.WriteLine('Missing $env:ChocolateyInstall environment variable')
  exit 1
}
```